zk_shell/shell.py

```diff
@@ -2778,6 +2778,35 @@ def complete_reconfig(self, cmd_param_text, full_cmd, *rest):
         completers = [complete_cmd, complete_arg, complete_config]
         return complete(completers, cmd_param_text, full_cmd, *rest)
 
+    @ensure_params(Required("fmtstr"), MultiOptional("cmds"))
+    def do_echo(self, params):
+        """
+\x1b[1mNAME\x1b[0m
```

Comment:
I don't think extrapolation works for docstrings:

```
>> def bar():
    """
    i am bar
    """
    pass
>> print bar.__doc__
'\n    i am bar\n    '
>>def foo():
    """
    i am %s
    """ % ('foo')
    pass
>> print foo.__doc__
(empty)
```
Is this not expected or am I doing it wrong?